src/quacc/recipes/mlp/_base.py

```diff
@@ -64,6 +65,16 @@ def pick_calculator(
             kwargs["default_dtype"] = "float64"
         calc = mace_mp(**kwargs)
 
+    elif method.lower() == "sevennet":
+        from sevenn import __version__
+        from sevenn.sevennet_calculator import SevenNetCalculator
+
+        if "model" not in kwargs:
+            kwargs["model"] = "7net-0"
+        if "device" not in kwargs:
+            kwargs["device"] = "cpu"
```

Comment:
The default device is "auto". Is there a reason to override this default by forcing CPU?